backend/controllers/orderController.js

```
@@ -130,10 +187,28 @@
+    // Reverse the page logic: page 1 starts from the end
+    const effectivePage = totalPages - parseInt(page) + 1;
+    const skip = Math.max((effectivePage - 1) * limit, 0);
+    const orders = await orderModel.find({}).skip(skip).limit(parseInt(limit));
 
-    res.json({ success: true, orders });
+    res.json({
+      success: true,
+      orders,
+      totalOrders,
+      totalPages,
+      currentPage: parseInt(page),
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)